appengine/analytics/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- [START_EXCLUDE] -->
```

Comment:
Do you know this works?  I haven't managed to get it to kick in properly, it really needs a [START app] ... [END app]